### PR TITLE
move surfaceCheck files out of test subdir

### PR DESCRIPTION
**Pull request by rlcee**
surfaceCheck is a standard validation procedure, so its files should not be in test subdirs
- copy genconfig file to EventGenerator/defaultConfigs
- move geom files to Mu2eG4/geom
- update surfaceCheck fcl

### CODE REVIEW

File: Mu2eG4/fcl/surfaceCheck.fcl [hunk 1]
```diff
@@ -33,8 +33,8 @@ physics.producers.g4run.physics.physicsListName: "Minimal"
 services.SeedService.baseSeed         :  8
 services.SeedService.maxUniqueEngines :  20
 
-physics.producers.generate.inputfile : "Offline/Mu2eG4/test/genconfig_01.txt"
+physics.producers.generate.inputfile : "EventGenerator/defaultConfigs/surfaceCheck.txt"
```
Comment (kutschke) [inline Mu2eG4/fcl/surfaceCheck.fcl:36]: Does this need a leading Offline/ ?

Comment (rlcee) [reply Mu2eG4/fcl/surfaceCheck.fcl]: hoist by my own petard, fixed

**Comment by kutschke** [comment]
I plan to merge this after I merge the art upgrade.  Is that Ok with you?  I am leaving for dinner out in a few minutes so I will merge after I am back.

**Comment by rlcee** [comment]
> I plan to merge this after I merge the art upgrade. Is that Ok with you? I am leaving for dinner out in a few minutes so I will merge after I am back.

Anytime is fine.  If I see it go in, I'll set the nightly to remove test dirs before building

**Comment by kutschke** [comment]
@FNALbuild run build tests

**Comment by kutschke** [comment]
I just did the merge of Mackenzie’s PR and relaunched your tests.  If all goes well I will merge in about an hour.  Does that work for you?


From: Ray Culbertson ***@***.***>
Reply-To: Mu2e/Offline ***@***.***>
Date: Friday, July 23, 2021 at 5:13 PM
To: Mu2e/Offline ***@***.***>
Cc: Rob Kutschke ***@***.***>, Assign ***@***.***>
Subject: Re: [Mu2e/Offline] move surfaceCheck files out of test subdir (#537)


I plan to merge this after I merge the art upgrade. Is that Ok with you? I am leaving for dinner out in a few minutes so I will merge after I am back.

Anytime is fine. If I see it go in, I'll set the nightly to remove test dirs before building

**Review by kutschke — APPROVED** [review]
Looks Ok to me.

**Pull request merged**